pandas/core/arrays/integer.py

```diff
@@ -196,7 +196,10 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
             mask = mask.copy()
         return values, mask
 
-    values = np.array(values, copy=copy)
+    values = np.asarray(values, dtype=getattr(values, "dtype", object))
+    if copy:
```

Comment:
you only need to copy if values *has* a dtype (otherwise ``np.assarray`` by definition will copy)

Reply:
np.asarray returns `TypeError: data type not understood` when dtypes are pandas dtypes(i.e, Int8Dtype(),Int16Dtype(), Int64Dtype(), UInt8Dtype(), etc). 
See,
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=26801&view=logs&j=bef1c175-2c1b-51ae-044a-2437c76fc339&t=770e7bb1-09f5-5ebf-b63b-578d2906aac9&l=516

Reply:
Done